.gitlab/lassen.yml

```diff
@@ -15,43 +15,19 @@
   tags:
     - shell
     - lassen
-  variables:
-    PLAT: lassen
-
-# Build MFEM
-build_mfem_ser_lassen:
-  extends: [.with_gcc_8_3_1, .on_lassen]
-  needs: [setup]
-  stage: lassen_build
-  script:
-    - mkdir -p ${BUILD_PATH}
-    - cp -r ${CI_PROJECT_DIR} ${BUILD_PATH}/${CI_PROJECT_NAME}_lassen_ser
-    - cd ${BUILD_PATH}/${CI_PROJECT_NAME}_lassen_ser
-    - lalloc 1 -W 5 -q pdebug make -j cuda CUDA_ARCH=sm_70
+  rules:
+    - if: '$CI_COMMIT_BRANCH =~ /_lnone/ || $ON_LASSEN == "OFF"' #run except if ...
```

Comment:
When is `$CI_COMMIT_BRANCH` going to be `/_lnone/`?

Reply:
This is just a way for me, or developers, not to trigger the tests on lassen if for some reason they don't want to:
Just append "_lnone" to your branch name.

Reply:
In that case, shouldn't the pattern be `.*_lnone` instead of `/_lnone/`?

Reply:
It works well that way. Plus, if I don't want any CI, I can say "_qnone_lnone".

Reply:
Isn't this parsed in bash with `[[ condition ]]`? I don't think `/_lnone/` defines a pattern (that matches strings containing `_lnone`) in bash.

Reply:
For reference:
https://docs.gitlab.com/ee/ci/variables/README.html#syntax-of-cicd-variable-expressions
following RE2 regular expression syntax (https://github.com/google/re2/wiki/Syntax).

Reply:
I see. I didn't expect that.